src/libclang/type_parser.cpp

```diff
@@ -465,8 +465,13 @@ CXCursor get_instantiation_template(const CXCursor& cur, const CXType& type,
     }
 }
 
+#ifndef CPPAST_TEMPLATE_FULLARGUMENTSPARSING
```

Comment:
Just cast the context parameter to void in the body instead of wrapping the signature in an #ifndef.